protocols/swap/test/Swap.js

```diff
@@ -81,9 +89,22 @@ contract('Swap', async accounts => {
     it('Deployed test contract "DAI"', async () => {
       tokenDAI = await FungibleToken.new()
     })
+
+    it('Set up TokenRegistry', async () => {
```

Comment:
Add tests for when swapping tokens with a `kind` that isn't in the token registry

Reply:
good catch!